pennylane/devices/default_qubit.py

```diff
@@ -66,6 +66,14 @@ def stopping_condition(op: qml.operation.Operator) -> bool:
         return True
     if op.__class__.__name__[:3] == "Pow" and qml.operation.is_trainable(op):
         return False
+    if (
+        isinstance(op, qml.operation.Operator)
+        and not isinstance(op, SymbolicOp)
```

Comment:
Why? I'm a little confused by this line.  We should be able to handle something like

```
qml.adjoint(qml.QubitUnitary(csr_mat, wires=0))
qml.ctrl(qml.QubitUnitary(csr_mat, 0), (1,2))
```
just fine right out of the box.

Reply:
co-resolved in a later suggestion